### PR TITLE
Adding header files to HEADLESS on Runtime.c to fix build on wince.

### PR DESCRIPTION
**Pull request by ricardobna**
## Description:
Removing some C files from wince build.

### Related Issue:
 Fixing this to avoid a furute issue with WinCE. 

## Motivation and Context:
The files removed are not compatible with WinCE build.

## Benefited Devices:
 - Device: All Window CE devices.
 - OS: Windows CE

## How Has This Been Tested?
 - Running the build process for Windows CE.

## Tested Devices:
 - Device: USA Windows Mobile 5.0 Pocket PC T2 Emulator
 - OS: Windos CE

### CODE REVIEW

File: TotalCrossVM/src/nm/lang/Runtime.c [hunk 1]
```diff
@@ -6,6 +6,9 @@
 #include "Runtime.h"
 #if defined(HEADLESS)
 #include "cpproc.h"
+#include <sys/types.h>
```
Comment (ItaloYeltsin) [inline TotalCrossVM/src/nm/lang/Runtime.c:9]: sys/types.h and errorno.h are inside and outside of the HEADLESS flag

Comment (ricardobna) [reply TotalCrossVM/src/nm/lang/Runtime.c:9]: Should be only inside the HEADLESS flag, I'll fix it

**Comment by ItaloYeltsin** [comment]
@acmlira what do you mean saying "linux block"?

File: TotalCrossVM/src/nm/lang/Runtime.c [hunk 2]
```diff
@@ -6,9 +6,10 @@
 #include "Runtime.h"
 #if defined(HEADLESS)
```
[the remark this answers is not in the document]
Comment (ricardobna) [reply TotalCrossVM/src/nm/lang/Runtime.c]: Did this and added it to some addtional files to better represent the platforms.

**Pull request merged**